query-engine/request-handlers/Cargo.toml

```diff
@@ -20,6 +21,8 @@ thiserror = "1"
 tracing = "0.1"
 url = "2"
 connection-string.workspace = true 
+quaint.workspace = true
```

Comment:
Why do we need quaint in request-handlers? It shouldn't be a dependency outside of sql-query-connector in the QE